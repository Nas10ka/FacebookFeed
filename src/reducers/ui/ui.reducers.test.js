//Instruments
import { Record } from 'immutable';
import reducer from './';

const initialState = new Record({
    initialized:  false,
    authFetching: false,
    feedFetching: false,
})();

describe('UI reducers:', () => {
    test('UI reducer initialized', () => {
        const next = reducer(initialState, { type: 'INITIALIZE' });

        expect(next).toMatchSnapshot();
    });
});
