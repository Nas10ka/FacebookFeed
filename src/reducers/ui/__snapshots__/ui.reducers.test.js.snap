// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI reducers: UI reducer initialized 1`] = `
Immutable.Record {
  "initialized": true,
  "authFetching": false,
  "feedFetching": false,
}
`;
